src/main/java/nl/tudelft/contextproject/gui/CameraFeedsController.java

```diff
@@ -0,0 +1,245 @@
+package nl.tudelft.contextproject.gui;
+
+import javafx.application.Platform;
+import javafx.beans.property.FloatProperty;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.control.ChoiceBox;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.image.WritableImage;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+import nl.tudelft.contextproject.ContextTFP;
+import nl.tudelft.contextproject.camera.Camera;
+
+import java.io.IOException;
+import java.util.Collection;
+import java.util.Iterator;
+
+
+
+public class CameraFeedsController {
+    
+    private static LiveStreamHandler leftStreamHandler;
+    private static LiveStreamHandler rightStreamHandler;
+    
+    @FXML private ChoiceBox<Camera> camChoiceOne;
+    @FXML private ChoiceBox<Camera> camChoiceTwo;
+    
+    @FXML private ImageView viewOne;
+    @FXML private ImageView viewTwo;
+    
+    @FXML private VBox streamBoxOne;
+    @FXML private VBox streamBoxTwo;
+    
+    private Collection<Camera> cameras;
+    
+    /**
+     * Initialize method used by JavaFX.
+     */
+    @FXML private void initialize() {
+        cameras = Camera.getAllCameras();
+        
+        leftStreamHandler = new LiveStreamHandler();
+        rightStreamHandler = new LiveStreamHandler();
+        
+        initChoiceBoxes();
+        initStreams();        
+        
+        viewOne.setImage(loadImage("black.png"));
+        viewTwo.setImage(loadImage("black.png"));
+    }
+    
+    /**
+     * Initializes the choice boxes.
+     * Allows the user to choose his camera feed for each view.
+     */
+    private void initChoiceBoxes() {
+        ObservableList<Camera> choices = FXCollections.observableArrayList();
+        choices.add(Camera.DUMMY);
+        Iterator<Camera> it = cameras.iterator();
+        while (it.hasNext()) {
+            choices.add(it.next());
+        }
+
+        camChoiceOne.setItems(choices);
+        camChoiceTwo.setItems(choices);
+        
+        camChoiceOne.setValue(Camera.DUMMY);
+        camChoiceTwo.setValue(Camera.DUMMY);
+        
+        addChoiceListener(camChoiceOne);
+        addChoiceListener(camChoiceTwo);
+    }
+    
+    private void initStreams() {        
+        addStreamListeners();
+    }
+    
```

Comment:
Why is this a method? Only having one statement in a method kinda doesn't make any sense